Source/WebCore/page/scrolling/ScrollAnchoringController.h

```diff
@@ -30,23 +30,25 @@
 
 namespace WebCore {
 
-class Element;
+class RenderBox;
 
 class ScrollAnchoringController final {
     WTF_MAKE_FAST_ALLOCATED;
 public:
     explicit ScrollAnchoringController(FrameView& frameView)
         : m_frameView(frameView)
     { }
-    void invalidateAnchorElement();
+    void invalidateAnchorBox();
     void updateScrollPosition();
-    void selectAnchorElement();
+    void chooseAnchorBox();
     FrameView& frameView() { return m_frameView; }
 
 private:
+    FloatSize computeOffset(RenderBox* candidate);
+
     FrameView& m_frameView;
-    WeakPtr<Element> m_anchorElement;
-    FloatPoint m_lastPositionForAnchorElement;
+    WeakPtr<RenderBox> m_anchorBox;
+    FloatSize m_lastPositionForAnchorBox;
```

Comment:
Generally we'd use `FloatPoint` for a position, and `FloatSize` for an offset (or delta).

Reply:
This is technically an offset, so changed the name to reflect that.